manifest.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "obsidian-spaced-repetition",
     "name": "Spaced Repetition",
-    "version": "1.10.1",
+    "version": "1.11.0-beta.0",
```

Comment:
For now, let's not bump this just yet. Plugin downloads won't work since Obsidian checks this file to find the release

Reply:
Hi Stephen, I've restored the version number as requested.